tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/dav/CalDavEventAttendanceRepository.java

```diff
@@ -99,6 +107,32 @@ private Mono<EventUid> getCalendarEventUidByBlobId(DavUser davUser, MessagePartB
             .map(EventUid::fromMessageHeaders);
     }
 
+    public Mono<Boolean> freeBusyQuery(DavUser davUser, EventUid eventUid, CalendarEventParsed calendarEventParsed) {
```

Comment:
`public Mono<Boolean> freeBusyQuery`

It is hard to understand if `true` is free or busy if I just read the method name. Maybe rename the method name?

Reply:
+1 for an explicit enum